src/yb/util/errno.h

```diff
@@ -47,6 +48,18 @@ inline std::string ErrnoToString(int err) {
   return std::string(buf);
 }
 
+inline void LogFatalForFallocateFailure(int alloc_status, int err) {
+  if (alloc_status < 0) {
+    if (err == EOPNOTSUPP) {
+      YB_LOG_FIRST_N(FATAL, 1) << "The filesystem does not support fallocate().";
+    } else if (err == ENOSYS) {
+      YB_LOG_FIRST_N(FATAL, 1) << "The kernel does not implement fallocate().";
+    } else {
+      YB_LOG_FIRST_N(FATAL, 1) << "fallocate() failed.";
```

Comment:
In the current code, STATUS_IO_ERROR() is returned.
Can LogFatalForFallocateFailure() return Status for this case (which is then returned in the caller) ?

Reply:
Yes, an error status was being returned and I believe that is also what was intended to be changed to fatal so that the YB scripts would fail fast, even in the case of unhandled errno from fallocate() failure. Do we really want to put that status error back?